SetupDependencies.ps1

```diff
@@ -10,7 +10,15 @@ ForEach ($RequiredModule In $RequiredModules)
     # Check to see if the module exists on the PSGallery
     ElseIf (Find-Module -Name:($RequiredModule))
     {
-        Install-Module -Name:($RequiredModule) -Force
+        Write-Host("[status]Installing module: '$RequiredModule' from 'PSGallery'")
+        if ($RequiredModule -eq 'PSScriptAnalyzer')
+        {
+            Install-Module -Name $RequiredModule -Repository:('PSGallery') -RequiredVersion '1.19.1' -Force
```

Comment:
Rather than setting a specific check here can we turn `$RequiredModules` into a hash table with `key=moduleName` and `value=version` and iterate though that?